### ISSUE TITLE
twisted.test.test_iutils fails on Windows with Python 3.5

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8758]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8758|
|Type|enhancement|
|Created|2016-08-14 14:35:05Z|
|Branch|https://github.com/twisted/twisted/tree/8758-rodrigc-iutils-py3|

When running twisted.test.test_iutils on Windows with Python 3.5, it fails:

```
===============================================================================
[FAIL]
Traceback (most recent call last):
  File "C:\Users\Craig\twisted3\twisted\trial\_synctest.py", line 425, in assertEqual
    super(_Assertions, self).assertEqual(first, second, msg)
  File "C:\Python35\Lib\unittest\case.py", line 820, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python35\Lib\unittest\case.py", line 813, in _baseAssertEqual
    raise self.failureException(msg)
twisted.trial.unittest.FailTest: b'hello world\r\n' != b'hello world\n'

twisted.test.test_iutils.ProcessUtilsTests.test_output
===============================================================================
[FAIL]
Traceback (most recent call last):
  File "C:\Users\Craig\twisted3\twisted\test\test_iutils.py", line 123, in gotOutputAndValue
    self.assertEqual(out, b"hello world!\n")
  File "C:\Users\Craig\twisted3\twisted\trial\_synctest.py", line 425, in assertEqual
    super(_Assertions, self).assertEqual(first, second, msg)
  File "C:\Python35\Lib\unittest\case.py", line 820, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Python35\Lib\unittest\case.py", line 813, in _baseAssertEqual
    raise self.failureException(msg)
twisted.trial.unittest.FailTest: b'hello world!\r\n' != b'hello world!\n'

twisted.test.test_iutils.ProcessUtilsTests.test_outputAndValue
-------------------------------------------------------------------------------
```

<details><summary>Searchable metadata</summary>

```
trac-id__8758 8758
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8758_rodrigc_iutils_py3 8758-rodrigc-iutils-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1471185305183441 1471185305183441
changetime__1471873386080428 1471873386080428
version__None None
owner__Craig_Rodrigues__rodrigc_____ Craig Rodrigues <rodrigc@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/497

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

changes look good. Please merge

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set owner to @rodrigc<br>@rodrigc set status to `closed`|
|-|-|

In changeset b3bf0d24f5c0bd6e7ea44369abff1d395ef18c53
```
#!CommitTicketReference repository="" revision="b3bf0d24f5c0bd6e7ea44369abff1d395ef18c53"
Merge pull request #497 from twisted/8758-rodrigc-iutils-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8758

Reviewer: Fix test_iutils.py on Windows Python 3
```
